examples/time-log-kopia-repository-server/README.md

```diff
@@ -0,0 +1,231 @@
+# Protecting TimeLog application
+
+This is an example of using Kanister to protect a simple TimeLog application. This application is contrived but useful for demonstrating Kanister's features. Every second it appends the current time to a log file.
+
+Note: This application is used in the Kanister tutorial, for a detailed walkthrough click [here](https://docs.kanister.io/tutorial.html#tutorial).
+
+### 1. Deploy the Application
+
+The following command deploys the example Time Log application in `time-log` namespace:
+
+```bash
+$ kubectl create namespace time-log
+
+# Create a deployment whose log we'll ship to s3 using Kanister functions (that would eventually use Kopia repository server)
+$ kubectl apply -f ./examples/time-log/time-logger-deployment.yaml -n time-log
+deployment "time-logger" created
+
+### 2. Protect the Application
+
+#### Create Repository Server CR
+
+Create Kopia Repository Server CR if not created already
+
+- Create Kopia Repository using S3 as the location storage
+
+```bash
+$ kopia --log-level=error --config-file=/tmp/kopia-repository.config \
+--log-dir=/tmp/kopia-cache repository create --no-check-for-updates \
+--cache-directory=/tmp/cache.dir --content-cache-size-mb=0 \
+--metadata-cache-size-mb=500 --override-hostname=mysql.app \
+--override-username=kanisterAdmin s3 --bucket=<s3_bucket_name> \
+--prefix=/repo-controller/ --region=<s3_bucket_region> \
+--access-key=<aws_access_key> --secret-access-key=<aws_secret_access_key> --password=<repository_password>
+```
+
+- Generate TLS Certificates and create TLS secret for Kopia Repository Server for secure communication between Kopia Repository Server and Client
+
+```bash
+$ openssl req -newkey rsa:2048 -nodes -keyout key.pem -x509 -days 365 -out certificate.pem
+
+$ kubectl create secret tls repository-server-tls-cert --cert=certificate.pem --key=key.pem -n kanister
+```
+
+- Create Location Secrets for Kopia Repository
+
+```bash
+# The following file s3_location_creds.yaml is a sample file for creating s3 credentials secrets. It contains the credentials for accessing the s3 bucket.
+$ vi s3_location_creds.yaml
+
+apiVersion: v1
+kind: Secret
+metadata:
+   name: s3-creds
+   namespace: kanister
+   labels:
+      repo.kanister.io/target-namespace: monitoring
```

Comment:
```suggestion
      repo.kanister.io/target-namespace: monitoring
```
Lets remove this for now, we are not using this label anywhere

Reply:
Done !